crates/ruff_dev/src/format_dev.rs

```diff
@@ -516,10 +533,21 @@ fn format_dir_entry(
         return Ok((Ok(Statistics::default()), file));
     }
 
-    let file = dir_entry.path().to_path_buf();
-    let options = options.to_py_format_options(&file);
+    let path = dir_entry.path().to_path_buf();
+    let mut options = options.to_py_format_options(&path);
+
+    let settings = resolver.resolve(&path, pyproject_config);
+    // That's a bad way of doing this but it's not worth doing something better for format_dev
+    if settings.line_length != LineLength::default() {
```

Comment:
Is this because you're trying to see if it's not the default? Why can't we just use `settings.line_length`? Because it will be wrong for projects in the formatter CI that use Black but not Ruff?

Reply:
yes, this only exists to have a (dummy) usage for the line length without breaking the formatter ecosystem checks

Reply:
Should `Settings.line_length` be an `Option<LineLength>` if that's something we want to distinguish?